IPython/core/completer.py

```diff
@@ -1182,13 +1186,11 @@ def magic_matches(self, text):
 
         return [cast_unicode_py2(c) for c in comp]
 
-    def magic_config_matches(self, text):
+    def magic_config_matches(self, line_buffer):
```

Comment:
nitpicking but changing a positional parameter name, is a change of API as it can be used a a keyword argument.